docs/source/api_reference/forecasting.rst

```diff
@@ -63,6 +63,17 @@ Exponential Smoothing
 
     AutoETS
 
+AUTO REGRESSION
```

Comment:
```suggestion
Auto-regressive models
```

Reply:
let's make it to `Auto-regressive AR-X(p)`.